pages/base_page.py

```diff
@@ -192,3 +209,30 @@ def click_super_search(self):
             self.selenium.find_element(*self._super_search_locator).click()
             from pages.super_search_page import CrashStatsSuperSearch
             return CrashStatsSuperSearch(self.testsetup)
+
+    class Footer(Page):
+
+        _browserid_login_locator = (By.CSS_SELECTOR, 'div.login a.browserid-login')
+        _browserid_logout_locator = (By.CSS_SELECTOR, 'div.login a.browserid-logout')
+
+        @property
+        def is_logged_out(self):
+            return self.selenium.find_element(*self._browserid_login_locator).is_displayed()
+
+        @property
+        def is_logged_in(self):
+            return self.selenium.find_element(*self._browserid_logout_locator).is_displayed()
+
+        def login(self, email, password):
```

Comment:
Hmm .. tell me what you think .. I purposely didn't go this route because later iterations of tests will need to get at the credentials when verifying account information.